src/components/Main/Main.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import CinemasPage from "../CinemasPage/CinemasPage";
+import UserPage from "../UserPage/UserPage";
+import EditCinema from "../EditCinema/EditCinema";
+import EditFilm from "../EditFilm/EditFilm";
+import EditSession from "../EditSession/EditSession";
+import EditUsers from "../EditUsers/EditUsers";
+import AdminPage from "../AdminPage/AdminPage";
+import { Switch, Route } from "react-router-dom";
```

Comment:
use logical grouping for imports:
- third party libraries
- components
- styles
- images